object-storage-oracle-cloud/build.gradle.kts

```diff
@@ -0,0 +1,13 @@
+plugins {
+    io.micronaut.build.internal.`objectstorage-module`
```

Comment:
If you want to avoid the weird syntax, you can use `id("io.micronaut.build.internal.objectstorage-module")`

Reply:
Yeah, unfortunately, I've been bitten by the type-safe and autocomplete monster!